<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FramePreview</class>
 <widget class="QWidget" name="FramePreview">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QCheckBox" name="actualSize">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>260</y>
     <width>91</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>Actual Size</string>
   </property>
  </widget>
  <widget class="QSlider" name="fpsSlider">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>260</y>
     <width>141</width>
     <height>21</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QSpinBox" name="fpsBox">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>260</y>
     <width>42</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="playPause">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>260</y>
     <width>41</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Play</string>
   </property>
  </widget>
  <widget class="QGraphicsView" name="graphicsView">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>401</width>
     <height>261</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>280</y>
     <width>49</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>FPS</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
